SH4RE/SH4RE/Account/AccountView.swift

```diff
@@ -166,6 +167,14 @@ struct AccountView: View {
                     .onAppear() {
                         getCurrentUser() {(result) in
                             name = result.name
+                            let storageRef = Storage.storage().reference(withPath: result.pfpPath)
+                            storageRef.getData(maxSize: 1 * 1024 * 1024) { [self] data, error in
```

Comment:
this doesnt break the preview right? I know I had to add the "as Int64" for viewlistingview, so just double checking!